cmd/controller-manager/app/controllermanager.go

```diff
@@ -164,7 +161,7 @@ func setupControllers(mgr controllerruntime.Manager, opts *options.Options, stop
 		}
 	}
 
-	if IsControllerEnabled("clusterStatus", controllers) {
+	if opts.IsControllerEnabled("clusterStatus") {
```

Comment:
I have one more question, should we use the name format used by Kubernetes:

```
All controllers: attachdetach, bootstrapsigner, cloud-node-lifecycle
```

ref: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

Reply:
@snowplayfire How do you think?

Reply:
I'm ok with both formats (Linux or camel).